Gruntfile.js

```diff
@@ -19,13 +19,13 @@ module.exports = function(grunt) {
       semVersion = isDefaultBranch ? version : version + semVersionSuffix;
 
   function option( name, def ) {
-    return grunt.option( name ) || teamcityProps[ name ] || def;
+    return grunt.option( name ) || teamcityProps[name] || def;
   }
 
   function createConcatOptions( ) {
     var options = {
       options: {
-        banner: '/* VERSION: ' + semVersion + ' */\n',
+        banner: '/* VERSION: ' + semVersion + ' */\r\n',
```

Comment:
This should perhaps be an environment newline, but it's better as `\r\n` because the files are concat'ed on a windows machine so this at least makes our distribution consistent